glue_jupyter/bqplot/scatter/layer_artist.py

```diff
@@ -270,15 +268,16 @@ def _update_visual_attributes(self, changed, force=False):
                             self.state.size * self.state.size_scaling,
                         )
                         self.scatter_mark.size = None
-                        self.scale_size_scatter.min = 0
-                        self.scale_size_scatter.max = 1
                     else:
-                        self.scatter_mark.default_size = int(self.state.size_scaling * 25)
-                        self.scatter_mark.size = ensure_numerical(
-                            self.layer[self.state.size_att].ravel()
-                        )
-                        self.scale_size_scatter.min = float_or_none(self.state.size_vmin)
-                        self.scale_size_scatter.max = float_or_none(self.state.size_vmax)
+                        self.scatter_mark.default_size = int(self.state.size_scaling * 7)
+                        s = ensure_numerical(self.layer[self.state.size_att].ravel())
+                        s = ((s - self.state.size_vmin) /
+                             (self.state.size_vmax - self.state.size_vmin))
+                        np.clip(s, 0, 1, out=s)
+                        s *= 0.95
+                        s += 0.05
```

Comment:
Those numbers (`size_scaling * 7`, `s` ranging from 0.05 to 1.0) are taken from comparison with the matplotlib defaults?
I was wondering if the division by 0 in

> scatter_density_mark.py:210: RuntimeWarning: invalid value encountered in divide
    normalized_counts = (self._counts - vmin) / (vmax - vmin)

was related, but that warning already pops up prior to this PR – I suppose that's bad user input really.

Reply:
Yeah, the 7 is a magic constant that I thought gave the closest match to the matplotlib layer artist - basically it's meant to serve a role similar what 30 does [here](https://github.com/glue-viz/glue/blob/main/glue/viewers/scatter/layer_artist.py#L428).

Reply:
If the parent `LayerArtist` is crafted that way, matching that makes sense. I'm good with the implantation so far, let's wait a little if @astrofrog has any comments.